Datasets/WordSeg/WordSegDataset.swift

```diff
@@ -15,28 +15,51 @@
 import Foundation
 import ModelSupport
 
+/// A collection of raw and processed text used for training and validation
+/// of word segmentation models.
 public struct WordSegDataset {
+  /// A collection of text used for training.
   public let training: [WordSegRecord]
```

Comment:
```suggestion
  public let trainingText: [WordSegRecord]
```

or 

```suggestion
  public let trainingData: [WordSegRecord]
```

I realize this commit is about the comments, but one of the main functions of writing comments is that it informs naming and design.  In this case, it seems like saying what it is in the comment tells you what it should be called.  Calling this property "training" implies it is a representation of the process of training.

Reply:
Renamed `WordSegRecord` to `Phrase` and `training` to `trainingPhrases`.